pkg/tecdsa/signing/gen/pb/message.proto

```diff
@@ -26,4 +26,10 @@ message TSSRoundThreeMessage {
     uint32 senderID = 1;
     bytes payload = 2;
     string sessionID = 3;
+}
+
+message TSSRoundFourMessage {
+    uint32 senderID = 1;
+    bytes payload = 2;
+    string sessionID = 3;
 }
```

Comment:
I would consider adding a newline at the end of the file (it's a convention on unix-like systems).

Reply:
Fixed after https://github.com/keep-network/keep-core/pull/3252 was merged to `main`.